File: packages/insomnia-httpsnippet/test/fixtures/output/ruby/native/query.rb
require 'uri'
require 'net/http'

url = URI("http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value")

http = Net::HTTP.new(url.host, url.port)

request = Net::HTTP::Get.new(url)

response = http.request(request)
puts response.read_body

File: packages/insomnia-httpsnippet/test/fixtures/output/node/native/query.js
var http = require('http');

var options = {
  method: 'GET',
  hostname: 'mockbin.com',
  port: null,
  path: '/har?foo=bar&foo=baz&baz=abc&key=value',
  headers: {}
};

var req = http.request(options, function(res) {
  var chunks = [];

  res.on('data', function(chunk) {
    chunks.push(chunk);
  });

  res.on('end', function() {
    var body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.end();
